src/main/scala/sbtdocker/DockerPush.scala

```diff
@@ -44,5 +47,20 @@ object DockerPush {
     val process = Process(command)
     val exitValue = process ! processLog
     if (exitValue != 0) sys.error("Failed to push")
+
+    val PushedImageId = ".* sha256:([0-9a-f]+) .*".r
```

Comment:
It seems that Docker documentation refers to the digest with `sha256:` as a prefix. Shouldn't it therefore be included here in the `ImageDigest` output?

Documentation for `ImageDigest` should then be updated.

Reply:
Yes, I think so, we can do something like this:

```
/**
  * The image digest, the format of the digest is `algorithm:hex-string`
  * @param digest A SHA 256 digest as a string.
  * @param algorithm The algorithm used to produce the digest.
  */
final case class ImageDigest(digest: String, algorithm: String = "sha256") {
  override def toString = s"$algorithm:$digest"
}
```